schemas/context/implementation.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/implementation",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Implementation",
```

Comment:
It probably makes sense to split this into a separate PR. Reviewing the extension point and the extension in one go is a bit hard.

Reply:
Usage and the used schemas are pretty interlinked.